cmd/internal/cli/actions.go

```diff
@@ -210,6 +211,32 @@ func setVM(cmd *cobra.Command) {
 	}
 }
 
+// returns url for library and sets auth token based on remote config
+// defaults to https://library.sylabs.io
+func handleActionRemote(cmd *cobra.Command) (remoteURL string) {
+	remoteURL = "https://library.sylabs.io"
+
+	// if we can load config and if default endpoint is set, use that
+	// otherwise fall back on regular authtoken and URI behavior
+	e, err := sylabsRemote(remoteConfig)
+	if err == scs.ErrNoDefault {
+		sylog.Warningf("No default remote in use, falling back to %v", remoteURL)
+		return
+	} else if err != nil {
+		sylog.Fatalf("Unable to load remote configuration: %v", err)
+	}
+
+	authToken = e.Token
+	uri, err := e.GetServiceURI("library")
```

Comment:
This is probably better written as:

```
remoteURL, err := e.GetServiceURI("library")
if err != nil {
    sylog.Warning("Unable to get library service URI: %v", err)
}
return remoteURL // or return remoteURL, err if you modify the return value as suggested
```

Reply:
This doesn't quite work since `GetServiceURI()` returns `""` in the case of an error and we want to always return a correct `URI` from this function. During the refactoring for the previous comment I think I address this.